package/jestEnv.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const NodeEnvironment = require("jest-environment-node").TestEnvironment;
+
+class SkiaEnvironment extends NodeEnvironment {
+  constructor(config, context) {
+    super(config, context);
+  }
+
+  async setup() {
+    await super.setup();
+    const {
+      LoadSkiaWeb,
+    } = require("@shopify/react-native-skia/lib/commonjs/web/LoadSkiaWeb");
+    this.global.CanvasKit = await LoadSkiaWeb();
```

Comment:
note that Jest also supports test envs written in native ESM, so you could use top level await and rely on module caching as the mutex for you